sources/packages/backend/apps/queue-consumers/test/helpers/mock-utils/cas-service.mock.ts

```diff
@@ -0,0 +1,72 @@
+import { CASService } from "../../../src/services/cas-supplier/cas.service";
+
+export const casLogonMockedResult = {
+  access_token: "token123",
+  token_type: "123456789",
+  expires_in: 3600,
+};
+
+export const getSupplierInfoFromCASMockedResult = {
```

Comment:
Would this not be the case for creating a factory that would allow the payload generation in different ways?
It can be created as simply as this PR requires and later be expanded, for instance, to add multiple addresses or have its values generated from using faker lib.